integrations/docker/images/chip-build-imx/Dockerfile

```
@@ -0,0 +1,17 @@
+ARG VERSION=latest
+FROM connectedhomeip/chip-build:${VERSION} as build
+
+RUN set -x \
+    && cd /opt/ \
+    && wget --quiet -O fsl-l5.10.52-2.1.0-sdk.tar.gz https://www.nxp.com/lgfiles/IMM/fsl-l5.10.52-2.1.0-sdk.tar.gz \
+    && tar zxvf fsl-l5.10.52-2.1.0-sdk.tar.gz \
+    && ./fsl-imx-xwayland-glibc-x86_64-imx-image-core-cortexa53-crypto-imx8mmevk-toolchain-5.10-hardknott.sh -y \
+    && : # last line
+
+WORKDIR /opt
```

`WORKDIR` Docker instruction define the working directory so moving that instruction before the `RUN` instruction avoid issues in [Hadolint](https://github.com/hadolint/hadolint)

```suggestion
WORKDIR /opt
RUN set -x \
    && wget --quiet -O fsl-l5.10.52-2.1.0-sdk.tar.gz https://www.nxp.com/lgfiles/IMM/fsl-l5.10.52-2.1.0-sdk.tar.gz \
    && tar zxvf fsl-l5.10.52-2.1.0-sdk.tar.gz \
    && ./fsl-imx-xwayland-glibc-x86_64-imx-image-core-cortexa53-crypto-imx8mmevk-toolchain-5.10-hardknott.sh -y \
    && : # last line


```

---

Hi, Thank you for the comments. Already add the "-imx" image support in docker_img.yaml.
Also the working dir instruction moved as the suggestion.